CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs

```diff
@@ -31,6 +33,8 @@ public BrowserTabView()
 
             //browser.BrowserSettings.BackgroundColor = Cef.ColorSetARGB(0, 255, 255, 255);
 
+            browser.WpfKeyboardHandler = new WpfImeKeyboardHandler(browser, Colors.Transparent);
+
```

Comment:
Please at a minimum comment this out, add a comment saying uncomment to enable experimental IME handling.

Reply:
Added comments. :)